content/docs/05.developer-guide/03.scripts.md

```diff
@@ -0,0 +1,800 @@
+- **Support for complex use cases**: the `Script` task is not meant for complex use cases where your business logic is comprised of multiple files importing classes and functions from other modules, etc.
+
+Overall, we recommend using `git.Clone` and `Commands` tasks for advanced production workloads. However, the `Script` tasks offer a great alternative for simple use cases.
+
+---
+
+## Runners
+
+You can configure your scripts to run either in local **processes** or in **Docker containers** by using the `runner` property:
+
+1. By default all scripting tasks run in isolated containers using the `DOCKER` runner. 
+2. Setting the `runner` property to `PROCESS` will execute your task in a local process on the Worker without relying on Docker for container isolation.
+
+### runner: DOCKER
+
+This is the default option for all script tasks. There are many arguments that can be provided here, including credentials to private Docker registries:
+
+```yaml
+id: pythonInContainer
+namespace: dev
+
+tasks:
+  - id: wdir
+    type: io.kestra.core.tasks.flows.WorkingDirectory
+    tasks:
+      - id: cloneRepository
+        type: io.kestra.plugin.git.Clone
+        url: https://github.com/kestra-io/examples
+        branch: main
+
+      - id: gitPythonScripts
+        type: io.kestra.plugin.scripts.python.Commands
+        warningOnStdErr: false
+        commands:
+          - python scripts/etl_script.py
+        runner: DOCKER
+        docker:
+          image: annageller/kestra:latest
+          config: |
+            {
+              "auths": {
+                  "https://index.docker.io/v1/": {
+                      "username": "annageller",
+                      "password": "{{secret('DOCKER_PAT')}}"
+                  }
+              }
+            }
+      - id: output
+        type: io.kestra.core.tasks.storages.LocalFiles
+        outputs:
+          - "*.csv"
+          - "*.parquet"
+```
+
+
+### runner: PROCESS
+
+The `PROCESS` runner is useful if your Kestra instance is running [locally without Docker](../09.administrator-guide/02.deployment/03.manual.md) and you want to access your local files and environments, for example, to take advantage of locally configured Conda virtual environments. 
+
+Running scripts in a local process is particularly beneficial when using [remote Worker Groups](../08.architecture.md#worker-group-ee). The example below ensures that a script will be picked up only by Kestra workers that have been started with the key `gpu`, effectively delegating processing of scripts that require demanding computational requirements to the right server, rather than running them directly in a local container:
+
+```yaml
+id: gpuTask
+namespace: dev
+
+tasks:
+  - id: hello
+    type: io.kestra.plugin.scripts.python.Commands
+    runner: PROCESS
+    commands:
+		- python ml_on_gpu.py
```

Comment:
There is a strange rendered stuff here, can you check that you didn't had a special character ?